README.md

```diff
@@ -318,7 +330,7 @@ Before making a new plugin release, ensure code is in high quality, fully tested
 
 1. Update your `~/.m2/settings.xml` according to the [Jenkins docs](https://wiki.jenkins.io/display/JENKINS/Hosting+Plugins#HostingPlugins-Releasingtojenkins-ci.org).
 2. Setup and run a GitHub [ssh agent](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/#adding-your-ssh-key-to-the-ssh-agent).
-3. Run `mvn release:prepare release:perform -B` from the HEAD of master
+3. Run `mvn release:prepare release:perform -B` from the HEAD of main
```

Comment:
Has the default branch of this repo been renamed to `main` and `main` branched off `master`, with `master` being removed?

Reply:
That's next up. There is also a reference in the badge that I meant to rename.